tools/validate/validate.cpp

```diff
@@ -191,7 +193,19 @@ int main() {
             continue;
         }
 
-        const string extension = rdi->path().extension().string();
+        const wstring& relative_path = filepath.native();
+
+        constexpr size_t maximum_relative_path_length = 120;
+        if (relative_path.size() > maximum_relative_path_length) {
+            fwprintf(stderr, L"Validation failed: the path \"%s\" is too long (%zu characters; the limit is %zu).\n",

```

Comment:
Should probably `%ls`. Ditto below.

Reply:
Ah, I didn't realize that was a way to select Standard behavior regardless of the weirdness here, thanks.